packages/pluggableWidgets/datagrid-web/src/Datagrid.editorConfig.ts

```diff
@@ -160,3 +171,17 @@ export const getPreview = (values: DatagridPreviewProps): StructurePreviewProps
         children: [headers, ...Array.from({ length: 5 }).map(() => columns), ...footer]
     };
 };
+
+export function check(values: DatagridPreviewProps): Problem[] {
+    const errors: Problem[] = [];
+    values.columns.forEach((column: ColumnsPreviewType) => {
+        if (!column.attribute && (column.sortable || values.columnsFilterable)) {
```

Comment:
Attribute in this case is always there because we do a check in editorConfig with Consistency check

Reply:
The check should be inside this function then. Now it will only throw an error when there is no attribute and the column is sortable or filterable. Not when `attribute` is selected for `showContentAs` and no attribute has been selected.

Reply:
Good catch, fixed :)